Makefile

```diff
@@ -29,15 +29,16 @@ install: earlyoom.service install-bin install-default
 install-initscript: earlyoom.initscript install-bin install-default
 	install -d $(DESTDIR)$(SYSCONFDIR)/init.d/
 	cp $< $(DESTDIR)$(SYSCONFDIR)/init.d/earlyoom
-	chmod a+x $(DESTDIR)$(SYSCONFDIR)/init.d/earlyoom
+	chmod 0755 $(DESTDIR)$(SYSCONFDIR)/init.d/earlyoom
 	-update-rc.d earlyoom start 18 2 3 4 5 . stop 20 0 1 6 .
 
 earlyoom.%: earlyoom.%.in
 	sed "s|:TARGET:|$(PREFIX)$(BINDIR)|g;s|:SYSCONFDIR:|$(SYSCONFDIR)|g" $< > $@
 
 install-default: earlyoom.default
 	install -d $(DESTDIR)$(SYSCONFDIR)/default/
-	install -m 644 $< $(DESTDIR)$(SYSCONFDIR)/default/
+	cp $< $(DESTDIR)$(SYSCONFDIR)/default/earlyoom
+	chmod 0644 $(DESTDIR)$(SYSCONFDIR)/default/earlyoom
```

Comment:
Don't understand yet. What's the improvement?

Reply:
This is the typo. It would install as '/etc/default/earlyoom.default' rather than '/etc/default/earlyoom' ... my fault :(

Reply:
I see! Could be written like this?
```
install -m 644 $< $(DESTDIR)$(SYSCONFDIR)/default/earlyoom
```